### ISSUE TITLE
Setup server logging

### ISSUE THREAD

**Issue by craigmacgregor** [NAVCoin/polymorph#80]
Setup framework for logging server errors.

- [x] setup writelog functions to format and capture logs
- [x] setup email notifications for critical logs
- [x] Send email on server start, notifying the server has started.
- [x] Email Time, Version Number, any other info we want to send on startup.

- [x] Add version to config.js

If we want to send emails via the backend we'll need a mailserver

https://www.npmjs.com/package/smtp-server

Use Gmail for now.

basically copy Logger.js

https://github.com/NAVCoin/navtech/blob/master/src/lib/Logger.js
